### ISSUE TITLE
java.lang.RuntimeException: No reader function for tag tailrecursion.boot.core/version

### ISSUE THREAD

**Issue by maacl** [mathias/gleam#3]
I realise that this is example is more than 2 years old, but I find it interesting and would like to run it.

Running `boot gleam-app` on OSX produces:

```
java.lang.RuntimeException: No reader function for tag tailrecursion.boot.core/version
                      ...
 clojure.core/read-string  core.clj: 3497
boot.util/read-string-all  util.clj:  207
          boot.main/-main  main.clj:  140
                      ...
         boot.App.runBoot  App.java:  231
            boot.App.main  App.java:  338
```

**Comment by maacl** [comment]
My bad, used boot2.

**Comment by mathias** [comment]
Yes, this likely needs to be updated for boot 2. I'll look into it at some
point.

On Saturday, October 17, 2015, Martin Clausen notifications@github.com
wrote:

> Closed #3 https://github.com/mathias/gleam/issues/3.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/mathias/gleam/issues/3#event-438308650.

## 

Matt Gauger
